operator/main.go

```diff
@@ -112,8 +115,11 @@ func init() {
 	flags.StringVar(&metricsAddress, "metrics-address", ":6942", "Address to serve Prometheus metrics")
 	flags.BoolVar(&synchronizeServices, "synchronize-k8s-services", true, "Synchronize Kubernetes services to kvstore")
 	flags.BoolVar(&synchronizeNodes, "synchronize-k8s-nodes", true, "Synchronize Kubernetes nodes to kvstore and perform CNP GC")
+	flags.BoolVar(&synchronizeIdentities, "synchronize-identities", true, "Synchronize CRD backed identities to kvstore")
```

Comment:
We should enable garbage collector if identity allocation is set to crd and add the `synchronize-identities` flag once we add the synchronization functionality

Reply:
oh, and it currently enables CRD GC if CRD is the allocation mode and the GC interval is != 0. The default is non-zero (i.e. it can be disabled if you set the interval to 0)